submissions/evgenii-del/friends-app/js/main.js

```diff
@@ -0,0 +1,146 @@
+let friendsArr = [];
+const main = document.querySelector(".js-main");
+const gender = document.querySelector(".js-gender");
+const age = document.querySelector(".js-age");
+const name = document.querySelector(".js-name");
+const reset = document.querySelector(".js-reset");
+let nameProp, ageProp, genderProp, searchProp;
+
+
+const createBlock = (friend) => {
+    const block = document.createElement("div");
+    block.classList.add("card");
+    block.innerHTML = `
+                <div class="card__img">
+                    <img src="${friend.picture.medium}" alt="avatar">
+                </div>
+                <div class="card__content">
+                    <div class="card__top">
+                        <p>${friend.name.first}</p>
+                        <p>${friend.name.last}</p>
+                    </div>
+                    <div>
+                        <p>Number:</p>
+                        <p>${friend.phone}</p>
+                    </div>
+                    <div class="card__bottom">
+                        <p>Age: ${friend.dob.age}</p>
+                        <p>Gender: ${friend.gender}</p>
+                    </div>
+                </div>
+        `;
+    return block;
+}
+
+const renderItem = (friends) => {
```

Comment:
`renderItem` usually suggests that you render one item out of a list, but in this case you render complete list